### DISCUSSION TITLE
Is there an alternate to ReactFlow's Node Origin?

### DISCUSSION THREAD

**Discussion by Mansehej** [bcakmakoglu/vue-flow#1201] [Q&A]
React Flow's [Node Origin](https://reactflow.dev/api-reference/react-flow#node-origin) feature allows us to control node positions based on a different origin than top-left. Is there an alternate to that in VueFlow?

If not, would appreciate some insight into the easiest way to implement that - I will try to create a pull request for the same.

**Comment by bcakmakoglu** [comment]
Alas, no, there is not and there's no "easy" way to do this atm.

Vue Flow will go some serious API changes and with 2.0.0 nodeOrigin will probably be introduced, so I'd recommend just being patient until then 😄
